plugins/prow-agent/scripts/otel_collector.py

```
@@ -0,0 +1,215 @@
+        for time_type, seconds in sorted(active_time.items()):
+            mins, secs = divmod(int(seconds), 60)
+            print(f"    {time_type}: {mins}m {secs}s")
+
+    if api_requests:
+        print(f"\n  API Requests: {len(api_requests)}")
+
+
+def main():
+    parser = argparse.ArgumentParser(description="OTLP HTTP/JSON collector for Claude Code telemetry")
+    parser.add_argument("--port-file", required=True, help="File to write the assigned port number to")
+    parser.add_argument("--log-file", default="/tmp/claude-otel.jsonl", help="JSONL output file")
```

Ruff (0.15.15)

[error] Probable insecure usage of temporary file or directory: "/tmp/claude-otel.jsonl"

(S108)